src/stylesheets/elements/_buttons.scss

```diff
@@ -1,181 +1,215 @@
+  &.sds-button--small {
+    height: units(3);
+    width: units(3);
+    padding: 0;
+    font-size: size('body', 1);
   }
-
-  &.usa-button--active,
-  &:active {
-    background-color: color('secondary-darker');
-    color: color('white');
-  }
-
-  &.usa-focus,
-  &:focus {
-    outline-offset: units(2px);
+  &.sds-button--big {
+    height: units(5);
+    width: units(5);
+    font-size: size('body', 8);
   }
 }
 
 .usa-button--outline {
   background-color: transparent;
-  box-shadow: $button-stroke color('primary');
+  box-shadow: none;
 
   &:visited {
-    color: color('primary');
+    color: color("primary");
   }
 
   &.usa-button--hover,
   &:hover {
     background-color: transparent;
-    box-shadow: $button-stroke color('primary-light');
+    box-shadow: none;
   }
 
   &.usa-button--active,
   &:active {
     background-color: transparent;
-    color: color('secondary-darker');
-    box-shadow: $button-stroke color('secondary-darker');
+    color: color("secondary-darker");
+    box-shadow: none;
   }
 
   &.usa-button--inverse {
     background-color: transparent;
-    box-shadow: $button-stroke color('primary');
-    color: color('white');
+    color: color("white");
+    box-shadow: none;
 
     &:visited {
-      color: color('base-lighter');
+      color: color("base-lighter");
+      box-shadow: none;
     }
 
     &.usa-button--hover,
     &:hover {
       background-color: transparent;
-      box-shadow: $button-stroke color('base-lightest');
-      color: color('base-lightest');
+      color: color("base-lightest");
+      box-shadow: none;
     }
 
     &.usa-button--active,
     &:active {
       background-color: transparent;
-      box-shadow: $button-stroke color('white');
-      color: color('base-light');
+      color: color("base-light");
+      box-shadow: none;
+    }
+
+    &:disabled {
+      background-color: transparent;
+      color: color("base-light");
+      box-shadow: none;
     }
   }
 }
-
 .usa-button--base {
-  color: color('gold-70');
-  background-color: color('accent-warm-lighter');
-  box-shadow: $button-stroke color('base-light');
+  color: color("gold-70");
+  background-color: color("accent-warm-lighter");
+  //box-shadow: $button-stroke color("base-light");
+  @include u-border("base-light");
 
   &.usa-button--hover,
   &:hover {
-    color: color('gold-70');
-    background-color: color('white');
-    box-shadow: $button-stroke color('base');
+    color: color("gold-70");
+    background-color: color("white");
+    //box-shadow: $button-stroke color("base");
+    @include u-border("base");
   }
 
   &.usa-button--active,
   &:active {
-    color: color('white');
-    background-color: color('base-dark');
+    color: color("white");
+    background-color: color("base-dark");
+    @include u-border("base-dark");
   }
 }
 
 .usa-button--secondary {
-  background-color: color('primary-lighter');
-  box-shadow: $button-stroke color('primary');
+  background-color: color("primary-lighter");
```

Comment:
the background should be our base color if we're calling it .usa-button--secondary. Use a different sds-button modifier such as light which could then be extended for usa-button:hover with a slight border color change

Reply:
Will be addressed in another ticket